docs/linux/reporting_kernel_bugs.md

```diff
@@ -52,61 +52,119 @@ If you can't figure out the right fix, but have some understanding of the bug, p
+### CVE assignment
 
-* There should ideally be no delay between reports to `security@kernel.org` and `linux-distros@vs.openwall.org`.
-* When working on the patch together with the `security@kernel.org` members and upstream maintainers, keep the linux-distros aware of the progress.
-* There should ideally be no delay between CVE description publication, distros' updates, upstream commit and notification to `oss-security@lists.openwall.com`. All of these should be on the same day, at worst.
-* The moment the issue is made public (e.g. patch is submitted upstream, CVE description published, etc.) it must be reported to `oss-security@lists.openwall.com` right away.
+kernel.org is now CNA (from [February 13, 2024](https://www.cve.org/Media/News/item/news/2024/02/13/kernel-org-Added-as-CNA)),
+therefore they have reserved CVE IDs for the kernel vulnerability bugs classified as CVE.
 
-A good example of an LPE announcement structure on `oss-security@lists.openwall.com` can be found [here](http://seclists.org/oss-sec/2016/q4/607), however the timeline doesn't look right there: public announcement should have occurred right after the patch was submitted to netdev.
+CVE assignments (sometimes rejections) are publicly announced in public list, see [linux-cve-announce](https://lore.kernel.org/linux-cve-announce) on which you can [subscribe](https://subspace.kernel.org/subscribing.html).
+
+CVEs are assigned after-the-fact with 1-2 week delay usually.
+> No hardware CVEs (SPECTRE-like CVEs come from others) (c) Greg K-H
+
+, e.g. bugs in drivers are Linux kernel related, hence they can be treated as CVEs, and HW bugs in CPU, NIC, other chips that affect Linux kernel, Windows, OSX etc.
+that are not caused due to Linux kernel code, should be handled by HW vendor.
+
+Averaging 55 CVEs per week (2024 Q3):
+> CVE assignment team is overly cautious and assign CVE numbers to any bugfix that they identify
+> This explains the seemingly large number of CVEs that are issued by the Linux kernel team.
+
+Kernel CVE assignment process is explained [here](https://www.kernel.org/doc/html/latest/process/cve.html), quoting the important paragraph:
+> If the CVE assignment team misses a specific fix that any user feels should have a CVE assigned to it,
+> please email them at <cve@kernel.org> and the team there will work with you on it.
+> Note that no potential security issues should be sent to this alias,
+> it is ONLY for assignment of CVEs for fixes that are already in released kernel trees.
+> If you feel you have found an unfixed security issue, please follow the [normal Linux kernel security bug reporting process](https://docs.kernel.org/process/security-bugs.html).
+
+Note that kernel.org assigns CVE in a very descriptive format (which files, versions are effected etc.).
+See [CVE-2024-50078](https://lore.kernel.org/linux-cve-announce/2024102936-CVE-2024-50078-cfe9@gregkh/T/#u) as an example.
+
+### Process
+
+![reporting kernel bugs](./reporting_kernel_bugs.png)
```

Comment:
I love the graph, thanks a lot for making it!

A couple of nits:
 - can you please add an instruction for generating it to the commit message?
 - there are two arrows going from the reporter to security@kernel.org and oss-security, but it is not clear from the picture at which point the letter to oss-security should be sent. If the blue numbers 1-4 on the picture correspond to steps 1-4 below, maybe the second arrow should be marked with the number 2?

Reply:
Thanks for the comments! I've updated the .md with syntax/grammar corrections, updated .drawio scheme and .png graph with the addressed comments. PTAL